lib/mail/message.rb

```diff
@@ -2135,6 +2135,14 @@ def find_attachment
       filename
     end
 
+    def inline_image?
+     image? && inline?
+    end
```

Comment:
`inline?` is a method on the Mail::Part subclass, so this'll break for a Mail::Message itself.

Could move this into Mail::Part and override the `attachment?` method, perhaps?

```ruby
# class Part < Message

def attachment?
  super || inline_image?
end

# …

def inline_image?
  # …
```

Reply:
Thanks for the feedback - I'll try to get this in next by next week!